# 
# Submitted-By: Robert Schwebel, 2005-04-06
# Committed-By: Robert Schwebel
#
# Error: 
#
# a) Makefile system tries to build bogus replace/Makefile
# b) Remove some files on 'make maintainer-clean'
#
# Description: 
#
# a) Revision 0.99.136 removed the replace/ directory from ipkg, but 
#    forgot to remove it in configure.ac. The first hunk fixes this. 
#
# b) 'make maintainer-clean' should be able to remove all autogenerated
#    files (which shouldn't be in cvs anyway). This patch adds these 
#    files to MAINTAINERCLEANFILES. 
#
# State:
# 
# Submitted to Carl Worth <cworth@handhelds.org> and 
# Steve Ayer <steven.ayer@compaq.com> by RSC (2005-04-06). 
#

diff -urN ipkg-0.99.144/configure.ac ipkg-0.99.144-ptx1/configure.ac
--- ipkg-0.99.144/configure.ac	2005-02-22 23:47:46.000000000 +0100
+++ ipkg-0.99.144-ptx1/configure.ac	2005-04-06 16:21:57.000000000 +0200
@@ -50,4 +50,4 @@
 AC_FUNC_VPRINTF
 AC_CHECK_FUNCS([memmove memset mkdir regcomp strchr strcspn strdup strerror strndup strrchr strstr strtol strtoul sysinfo utime])
 
-AC_OUTPUT(Makefile etc/Makefile replace/Makefile familiar/Makefile familiar/control familiar/control-unstripped familiar/libipkg-control  familiar/libipkg-dev-control libbb/Makefile libipkg.pc ipkg.h)
+AC_OUTPUT(Makefile etc/Makefile familiar/Makefile familiar/control familiar/control-unstripped familiar/libipkg-control  familiar/libipkg-dev-control libbb/Makefile libipkg.pc ipkg.h)
diff -urN ipkg-0.99.144/Makefile.am ipkg-0.99.144-ptx1/Makefile.am
--- ipkg-0.99.144/Makefile.am	2005-02-22 00:34:14.000000000 +0100
+++ ipkg-0.99.144-ptx1/Makefile.am	2005-04-06 16:29:24.000000000 +0200
@@ -76,6 +76,15 @@
 
 EXTRA_DIST = ipkg.c update-alternatives
 
+MAINTAINERCLEANFILES=		\
+	configure		\
+	Makefile.in		\
+	config.guess		\
+	config.sub		\
+	ltmain.sh		\
+	.Makefile.am.swp	\
+	aclocal.m4
+
 libipkg_la_SOURCES = $(ipkg_libcore_sources) $(ipkg_cmd_sources) $(ipkg_db_sources) \
 	       $(ipkg_util_sources) $(ipkg_list_sources)
 
